pkg/serializer/types.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// +build !orchestrator
+
+package serializer
+
+// ProcessMessageBody is a type alias for processes proto message body
+type ProcessMessageBody = stubMessageBody
```

Comment:
nit: for future reference, I think it's worth explaining in the comment why you went through the trouble of using a type alias here :)